contracts/modules/RelayerManager.sol

```diff
@@ -387,25 +370,24 @@ abstract contract RelayerManager is BaseModule {
         assembly { chainId := chainid() }
     }
 
-    function getRequiredSignaturesToStartSession(address _wallet) internal view returns (uint256, OwnerSignature) {
-        // owner  + [n/2] guardians
-        uint numberOfGuardians = Utils.ceil(guardianStorage.guardianCount(_wallet), 2);
-        require(numberOfGuardians > 0, "RM: no guardians set on wallet");
-        uint numberOfSignatures = 1 + numberOfGuardians;
-        return (numberOfSignatures, OwnerSignature.Required);
+    function startSession(address _wallet, address _sessionUser, uint64 _duration) external onlyWallet(_wallet) returns (bool)
```

Comment:
`return (bool)` is no longe necessary

Reply:
I'm probably missing something but why don't we make this method `internal` and change `multiCallWithGuardians` to:

```
function multiCallWithGuardians(address _wallet, Call[] calldata _transactions, Session calldata _session) external onlySelf() onlyWhenUnlocked(_wallet) returns (bytes[] memory)
{
        multiCallWithApproval(_wallet, _transactions);
        if(_session.key != address(0)) {
            startSession(_wallet, _session.key, _session.expires);
        }
 }
 ```
 in order to save the gas required for the Proxy->BaseWallet->ArgentModule calls?